### ISSUE TITLE
Identifying Future High Cost Individuals within an Intermediate  Cost Population

### ISSUE THREAD

**Issue by wanghaisheng** [e-plus-healthcare-alliance/Health-cost-prediction-models#1]
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4874657/pdf/nihms755930.pdf

**Comment by wanghaisheng** [comment]
>The goal should not only be to target the highest cost group but also aim interventions at 
preventing individuals from entering into this group in the first place. Predicting healthcare 
costs is an option to shape healthcare delivery programs and potentially improve health and 
control costs. Health cost prediction models often incorporate demographic information and 
information on clinical conditions based on data from medical records or claims databases 5.

Clinical conditions have been entered into prediction models as the diagnostic cost group (DCG), prevalent chronic conditions, or counts of chronic conditions. DCG models were originally developed to match HMO payments to the healthcare needs of enrollees. The system uses patients’ age, gender and medical diagnosis profiles to predict healthcare expenditures11–17. While these methods have been broadly used in Medicare databases, they are also applicable to private insurance and Medicaid databases 18. However, the system is limited by its requirement of special resources (e.g., commercial software and expertise), dependence on a common classification structure within ICD-9 codes, and accuracy of these codes.

As an alternative, the presence of certain chronic conditions can predict health expenditures in various settings 19–22. A third approach to predicting future costs is to use the count of 
chronic conditions. Several studies suggested that a simple count of chronic conditions can 
predict the length of hospital stay and mortality 23 and amount of health expenditures 19,22. 
The study by Farley et al 19 showed that not only the simple count of diagnose clusters but 
also the counts of prescriptions and physician visits were better predictors of future costs 
than the comorbidity measures (i.e. Charlson and Elixhauser indexes) with a count of diagnose clusters being the best predictor among all measurements examined. Similarly, Fleishman and Cohen 22, using data from the national Medical Expenses Survey, compared the ability of the DCG method, counts of chronic conditions, and the presence of ten prevalent chronic indicators to predict the top ten percent medical expenditure. The results showed that the count of chronic conditions significantly predicted future high-cost, controlled for DCG category, demographic characteristics and self-reported functional status.
To advance our understanding of how population management principles should guide the 
structure of care delivery, we sought to determine what factors correlate with transitioning to 
the highest cost segment of the population within the VCC program 10. Using demographic data, diagnosis information, and medication utilization of healthcare utilization, we describe the factors that are associated with the transition of individuals from an intermediate cost segment to a high cost segment of the population

**Comment by wanghaisheng** [comment]
Our study, by systematically comparing the ability of the count of chronic conditions, the ten 
most prevalent chronic indicators, counts of prescriptions, and counts of total hospital visits 
to predict future high-costs, had results consistent with other studies 19,22. Our prediction 
models, however, combined clinical and administrative data without the complexity of DCG 
scores. Hence, the results derived from these models may be easier to implement. Moreover, 
our study focused on an uninsured adult population whose income was under 200% FPL. 
Our results may provide unique insights relevant to the newly insured populations under 
Medicaid expansion.

Noticeably, the model discrimination presented by this study is lower (c=0.68) than the results presented by other studies (c ranged from 0.81–0.85) 22,26,27. This difference may be 
a consequence of several factors. Our model included the counts of chronic conditions and 
presence of individual chronic conditions as the main predictors of high costs. These indicators are sensitive to information on severity of the conditions. For example, a study by Omachi and colleagues 28 showed that adding COPD severity measures significantly improved predictions of medical costs in the following year among a cohort of patients with COPD. Adding further information regarding the severity of each condition may increase the predictive power of the model. Additionally, the overall prevalence of chronic conditions were relative low in our dataset, likely due to the population being younger (mean age=43.3 years, standard deviation=12.6 years). Leaders and researchers should select the model that best fits the demographics of the population of interest.

This study has several limitations. Our dataset may not have contained all data on utilization 
or diagnoses for each patient. In addition, similar to the cost prediction results drawn from 
other administrative databases, we are unable to adjust the heterogeneities of disease 
conditions in the cost prediction model. Finally, the patient population included in this study 
is medically underserved and they are relatively younger and healthier, thus the results may 
not be generalizable to other patient population segments, such as those more affluent or 
individuals with better access to care, and those with older age and sicker conditions. Further 
model validation studies are needed to confirm our findings.

In conclusion, the results from this study show that a simple model including demographics 
and health utilization indicators is associated with high future costs. Specific medical 
diagnoses, such as congestive heart failure, and the count of chronic conditions were also 
associated with higher future utilization. Further validation of the models is recommended to 
confirm the predictive capacity of this approach. If confirmed, this approach could be used 
to identify high-risk individuals and target interventions that decrease utilization and improve health.

**Comment by wanghaisheng** [comment]
References
1. Berwick DM, Nolan TW, Whittington J. The triple aim: care, health, and cost. Health Aff 
(Millwood). 2008; 27:759–769. [PubMed: 18474969] 
2. Lillrank P, Groop PJ, Malmstrom TJ. Demand and supply-based operating modes--a framework for 
analyzing health care service production. Milbank Q. 2010; 88:595–615. [PubMed: 21166870] 
3. Cohen S, Yu W. The concentration and persistence in the level of health expenditures over time: 
Estimates from the U. S. Population, 2008–2009. 2012
4. Lynn J, Straube BM, Bell KM, et al. Using population segmentation to provide better health care for 
all: the “Bridges to Health” model. Milbank Q. 2007; 85:185–208. discussion 209–12. [PubMed: 
17517112] 
5. Institute of Medicine. Crossing the qualty chasm: A new health system for the 21st century. 
Washington: National Academies Press; 2001. 
6. Eissens van der Laan MR, van Offenbeek MA, Broekhuis H, Slaets JP. A person-centred 
segmentation study in elderly care: towards efficient demand-driven care. Soc Sci Med. 2014; 
113:68–76. [PubMed: 24852657] 
7. Zhou YY, Wong W, Li H. Improving care for older adults: a model to segment the senior population. 
Perm J. 2014; 18:18–21. [PubMed: 24937151] 
8. Retchin SM, Garland SL, Anum EA. The transfer of uninsured patients from academic to 
community primary care settings. Am J Manag Care. 2009; 15:245–252. [PubMed: 19355797] 
Lu et al.
Page 8
Qual Prim Care
. Author manuscript; available in PMC 2016 May 20.
Author Manuscript
Author Manuscript
Author Manuscript
Author Manuscript
9. Bradley CJ, Gandhi SO, Neumark D, et al. Lessons for coverage expansion: a Virginia primary care 
program for the uninsured reduced utilization and cut costs. Health Aff (Millwood). 2012; 31:350–
359. [PubMed: 22323165] 
10. Dow AW, Bohannon A, Garland S, et al. The effects of expanding primary care access for the 
uninsured: implications for the health care workforce under health reform. Acad Med. 2013; 
88:1855–1861. [PubMed: 24128619] 
11. Ash, A.; Porell, F.; Gruenberg, L. Final Reportto the Health Care Financing Administration. 
Boston: Health Policy Research Consortium. Brandeis/Boston Universities; 1986. An Analysis of 
Alternative AAPCC Models Using Data from the Continuous Medicare History Sample. 
12. Ash A, Porell F, Gruenberg L, Sawitz E, Beiser A. Adjusting Medicare capitation payments using 
prior hospitalization data. Health Care Financ Rev. 1989; 10:17–29. [PubMed: 10313277] 
13. Ellis RP, Ash A. Refinements to the Diagnostic Cost Group (DCG) model. Inquiry. 1995; 32:418–
429. [PubMed: 8567079] 
14. Ellis, RP.; Pope, GC.; Iezzoni, LI. Final Report to the Health Care Financing Administration. Apr. 
1996 Diagnostic Cost Group (DCG) and Hierarchical Coexisting Conditions and Procedures 
(HCCP) Models for Medicare Risk Adjustmen. Contract No. 500-92-0020
15. Pope GC, Ellis RP and Liu CF. Final Report to the Health Care Financing Administration under 
Contract Number 500-95-048. Feb. 1998 Revised Diagnostic Cost Group (DCG)/Hierarchical 
Coexisting Conditions (HCC) Models for Medicear Risk Adjustment. 
16. Pope GC, Liu CF, Ellis RP. Principal Inpatient Diagnostic Cost Group Models for Medicaer Risk 
Adjustment. Final Report to the Health Care Financing Administration. Feb.1999 25:111–163.
17. Pope GC, Ellis RP, Ash AS, et al. Principal inpatient diagnostic cost group model for Medicare risk 
adjustment. Health Care Financ Rev. 2000; 21:93–118. [PubMed: 11481770] 
18. Ash AS, Ellis RP, Pope GC, et al. Using diagnoses to describe populations and predict costs. 
Health Care Financ Rev. 2000; 21:7–28. [PubMed: 11481769] 
19. Farley JF, Harley CR, Devine JW. A comparison of comorbidity measurements to predict 
healthcare expenditures. Am J Manag Care. 2006; 12:110–119. [PubMed: 16464140] 
20. Baser O, Palmer L, Stephenson J. The estimation power of alternative comorbidity indices. Value 
Health. 2008; 11:946–955. [PubMed: 18489502] 
21. Charlson ME, Charlson RE, Peterson JC, Marinopoulos SS, Briggs WM, Hollenberg JP. The 
Charlson comorbidity index is adapted to predict costs of chronic disease in primary care patients. 
J Clin Epidemiol. 2008; 61:1234–1240. [PubMed: 18619805] 
22. Fleishman JA, Cohen JW. Using information on clinical conditions to predict high-cost patients. 
Health Serv Res. 2010; 45:532–552. [PubMed: 20132341] 
23. Melfi C, Holleman E, Arthur D, Katz B. Selecting a patient characteristics index for the prediction 
of medical outcomes using administrative claims data. J Clin Epidemiol. 1995; 48:917–926. 
[PubMed: 7782800] 
24. Raftery AE. Bayesian Model Selection in Social Research Sociological Methodology. 25:111–163. 
995. 
25. Hosmer, DW.; Lemeshow, S. Applied Logistic Regression. New York: Wiley; 2000. 
26. Meenan RT, Goodman MJ, Fishman PA, Hornbrook MC, O’Keeffe-Rosetti MC, Bachman DJ. 
Using risk-adjustment models to identify high-cost risks. Med Care. 2003; 41:1301–1312. 
[PubMed: 14583693] 
27. DeSalvo KB, Jones TM, Peabody J, et al. Health care expenditure prediction with a single item, 
self-rated health measure. Med Care. 2009; 47:440–447. [PubMed: 19238099] 
28. Omachi TA, Gregorich SE, Eisner MD, et al. Risk adjustment for health care financing in chronic 
disease: what are we missing by failing to account for disease severity? Med Care. 2013; 51:740–
747. [PubMed: 23703646]

**Comment by wanghaisheng** [comment]
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4194677/pdf/hcfr-21-3-093.pdf
Principal Inpatient Diagnostic Cost Group Model for Medicare Risk Adjustment

https://www.cms.gov/Research-Statistics-Data-and-Systems/Statistics-Trends-and-Reports/Reports/downloads/pope_2000_2.pdf
Diagnostic Cost Group HierarchicalCondition Category Models for Medicare Risk Adjustment

**Comment by wanghaisheng** [comment]
http://www.sciencedirect.com/science/article/pii/S109830151060575X
http://ac.els-cdn.com/S109830151060575X/1-s2.0-S109830151060575X-main.pdf?_tid=ecef7918-8cad-11e7-bcd9-00000aab0f01&acdnat=1504006609_b34eced1e98439525f31ed487de5e0f6
The Estimation Power of Alternative Comorbidity Indices

**Comment by wanghaisheng** [comment]
Horizon Effects and Adverse Selection in Health Insurance Markets
https://pdfs.semanticscholar.org/0447/5a4cff3f6cbc70635acc553efbf706f16a49.pdf


The idea of extending contract length interacts with a number of issues related to health
insurance markets.First, two existing patterns of demand for insurance can influence the effect of contract length reform. Inertia in plan choice (as documented in Handel, 2013) implies that individ-
uals might be slow to upgrade their coverage after moving to two-year contracts. However,
one of the main results of the theoretical section is that the supply effect alone generates
more coverage, even in the case in which the demand curve stays fixed. This result implies
that inertia would reduce the efficiency gains of the reform, but not eliminate them.18 More-
over, there is some evidence that consumers have different valuations for insurance contracts
Bundorf et al., 2012), i.e., that there is some amount of horizontal differentiation. These
findings imply that contract length reform is particularly attractive as it preserves consumer
choice. In particular, the components of insurance contracts that display the most preference
heterogeneity (i.e., plan network) are largely orthogonal to horizon.
Another challenge to changing contract length in practice is commitment. In particu-
lar, moving to longer contracts involves restricting the possibility for individuals to change
plans after a year, even for such individuals that incur greater out-of-pocket costs as a
result. Currently, individuals can change their coverage around the year when any one of
several qualifying life events occurs (such as marriage or change of employment status). Such
exemptions could be abused to circumvent coverage change restrictions. While practical en-
forceability constraints are unlikely to be much more problematic for two-year contracts than
they are with one-year contracts, they are important when considering contracts that extend
further into the future. Handel et al. (2016) provides an in-depth discussion of long-term
contracts with imperfect commitment.

This paper shows how extending the horizon of health insurance contracts impacts adverse
selection in these markets. The main contribution of this paper is to show the implications of
the dynamics of health risk over time, as opposed to simply its cross-sectional distribution.
We show that increasing the contract horizon is another policy instrument that can be used to
reduce selection. Conceptually, we argue that private information is endogenous to contract
length because individual risk is harder to predict at longer horizons. This decrease in risk
predictability is strongly borne by the data. Estimating a model of ACA-like exchanges, we
find the effect of extending contracts to two years would be to expand coverage by about
6%. We also find positive, albeit moderate, welfare gains.

We study how increasing contract length affects adverse selection in health insur-
ance markets.  While health risks are persistent, private health insurance contracts
in the U.S. have short, one-year terms. Short-term, community-rated contracts allow
patients to increase their coverage only after risks materialize, leading to market unrav-
eling. Longer contracts ameliorate adverse selection because both demand and supply
exhibit horizon effects. Intuitively, longer horizon risk is less predictable, thus elevating
demand for coverage and lowering equilibrium premiums. We estimate risk dynamics
using data from 3.5 million U.S. health insurance claims and calculate counterfactual
coverage and welfare in equilibrium with two-year contracts.  We predict that such
contracts would increase coverage by 6% from its initial level and yield average annual
welfare gains of $100–$200 per person. Welfare gains from increased enrollment would
partly offset by exposing those with low coverage to greater risk.
